### PR TITLE
Aaron gandee - portfolio-website

### PR DESCRIPTION
**Pull request by AaronGandee**
final project submission

### CODE REVIEW

**Comment by AaronGandee** [comment]
Here is the live site:  https://aarongandee.github.io/#

File: README.md [hunk 1]
```diff
@@ -10,45 +10,45 @@ You have the ability to write HTML, CSS, and responsive media queries.  You also
 
 _ Watch this walk through video:  https://youtu.be/P2Y9W29kcjs
 
-- [ ] Create a forked copy of this project.
-- [ ] Add PM as collaborator on Github.
-- [ ] Clone your OWN version of the repository (Not Lambda's by mistake!).
-- [ ] Create a new branch on the clone: `git checkout -b <firstName-lastName>`.
-- [ ] Implement the project on the `<firstName-lastName>` branch, committing changes regularly.
-- [ ] Push commits: `git push origin <firstName-lastName>`.
+- [x] Create a forked copy of this project.
```
Comment (josenriagu) [inline README.md]: Good job!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+-->
+<html>
+	<head>
+		<title>Aaron Gandee Portfolio</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="stylesheet" href="assets/css/main.css" />
+		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
+	</head>
+	<body class="is-preload">
```
Comment (josenriagu) [inline index.html]: this **is-preload** class is the javascript responsible for slowing down your website. you may consider removing it entirely

File: index.html [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+
+				<!-- Header -->
+					<header id="header">
+						<div class="logo">
+							<span class="icon fa-gem"></span>
+						</div>
+						<div class="content">
+							<div class="inner">
+								<h1>Aaron Gandee</h1>
+								<p>I am a full stack web developer. I have a passion for technology and I love bringing peoples dreams of design to life!</p>
```
Comment (josenriagu) [inline index.html]: Beautiful!

**Pull request merged**